.travis.yml

```diff
@@ -43,8 +45,4 @@ install:
 script:
   # Usually, it's ok to finish the test scenario without reverting
   #  to the addon's original dependency state, skipping "cleanup".
-  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
-
-branches:
```

Comment:
Can you readd it?

Reply:
My bad, missed it when doing the diffing on `ember init`. Re-added.